src/com/company/M9/MyHashMap.java

```diff
@@ -0,0 +1,112 @@
+package com.company.M9;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+
+public class MyHashMap<K, V> {
+    private class Element<K, V> {
+        private K key;
+        private V value;
+
+        private Element(K key, V value) {
+            this.key = key;
+            this.value = value;
+        }
+
+        public K getKey() {
+            return key;
+        }
+
+        public V getValue() {
+            return value;
+        }
+
+        @Override
+        public String toString() {
+            return "Element{" +
+                    "key=" + key +
+                    ", value=" + value +
+                    '}';
+        }
+    }
+
+    private ArrayList<LinkedList<Element<K, V>>> buckets;
+    private int numBuckets;
+    private int size;
```

Comment:
two variables are not needed, `size`  will be enough. instead of `numBuckets` use `buckets.length()`